service/org.nemomobile.BtrfsBalancer.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.nemomobile.BtrfsBalancer
+Name=org.nemomobile.BtrfsBalancer
+Exec=/usr/sbin/btrfs-balancer server
+User=root
```

Comment:
It does not make it automatically start it makes it so that when someone calls it over dbus it can not start before all the other deps are ready. We we have the same thing for ssu though no After= lines atm. Currently if someone would call that dbus service file before filesystem is ready it would start, how early it can be called?